react/week-19/lesson.md

```diff
@@ -159,27 +157,28 @@ The last command should open a web browser for you. This shows the application t
 We looked at the beginning of the lesson at the concept of components. Now let's look at how components are made in React.
 
 ```js
-import React from 'react'
-import ReactDOM from 'react-dom'
+import React from 'react';
+import ReactDOM from 'react-dom';
 
 function HelloWorld() {
   return (
```

Comment:
Just a quick note on the parens here - I was deliberately trying to leave them in, because multi-line JSX can cause problems sometimes. 

I just tested this:

```js
return
  <div>
    <span>Hello world</span>
  </div>
```

Prettier will insert a semi after the return, which I think would trip up the students.

However this for this code:

```js
return <div>
  <span>Hello world</span>
</div>
```

Prettier does the right thing.  I feel like this would be the more common scenario for students to type, so in the end I don't think it really matters that much.

Overall I think it's something that's good to mention when showing a component for the first time.